release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2919,6 +2919,12 @@ It is anticipated that an upcoming z-stream release will include a fix for this
 +
 To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.(link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
 
+* Currently, the update to HAProxy version 2.6.13 in {product-title} {product-version} causes an increase in P99 latency for re-encrypt traffic. This is observed when the volume of ingress traffic puts the HAProxy component of the `IngressController` custom resource (CR) under a considerable load. The latency increase does not affect overall throughput, which remains consistent.
++
+The default `IngressController` CR is configured with 4 HAProxy threads. If you experience elevated P99 latencies during high ingress traffic conditions, specifically with re-encrypt traffic, it's recommended to increase the number of HAProxy threads to reduce latency. (link:https://issues.redhat.com/browse/OCPBUGS-18936[*OCPBUGS-18936*])
+
+* For Google Cloud Platform {sno-caps} to {product-version}, there is a known issue with the Cluster Network Operator getting stuck on a crash loop because the cloud network config controller recieves the wrong IP address from the name server when trying to reach the API server. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
```

Comment:
cc @ricky-rav : can you please help iron the details of our SNO+GCP+CNCC bug here and give a doc text for @JoeAldinger